db/schema.rb

```diff
@@ -37,14 +37,17 @@
     t.integer "height", null: false
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
+    t.bigint "root_id"
+    t.index ["root_id"], name: "index_flowcharts_on_root_id"
   end
 
   create_table "questions", force: :cascade do |t|
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
   end
 
-  add_foreign_key "flowchart_nodes", "flowchart_nodes", column: "child_id_id"
-  add_foreign_key "flowchart_nodes", "flowchart_nodes", column: "sibling_id_id"
-  add_foreign_key "flowchart_nodes", "flowcharts", column: "flowchart_id_id"
+  add_foreign_key "flowchart_nodes", "flowchart_nodes", column: "child_id"
+  add_foreign_key "flowchart_nodes", "flowchart_nodes", column: "sibling_id"
+  add_foreign_key "flowchart_nodes", "flowcharts"
```

Comment:
What does this line do? Just wondering

Reply:
My best guess is the column name is implicitly `tablename_id` so it does not write the column in the schema file. For the other columns we added, we haven't named them `tablename_id`, so it explicitly puts them.

I checked the schema for the `flowchart_nodes` table and it looks right. 
![image](https://user-images.githubusercontent.com/18633281/68241150-2dee7100-ffdc-11e9-971b-c82a1ee2c940.png)
![image](https://user-images.githubusercontent.com/18633281/68241162-33e45200-ffdc-11e9-8292-0d4be015876b.png)

Reply:
cool, looks good then